examples/material_gallery/test/smoke_test.dart

```diff
@@ -8,55 +8,122 @@ import 'package:test/test.dart';
 
 import '../lib/main.dart' as material_gallery;
 
+const List<String> demoCategories = const <String>['Demos', 'Components', 'Style'];
+const List<String> demoNames = const <String>[
+  'Weather',
+  'Fitness',
+  'Fancy lines',
+  'Flexible space toolbar',
+  'Floating action button',
+  'Buttons',
+  'Cards',
+  'Chips',
+  'Date picker',
+  'Data tables',
+  'Dialog',
+  'Drop-down button',
+  'Expand/collapse list control',
+  'Grid',
+  'Icons',
+  'Leave-behind list items',
+  'List',
+  'Menus',
+  'Modal bottom sheet',
+  'Over-scroll',
+  'Page selector',
+  'Persistent bottom sheet',
+  'Progress indicators',
+  'Scrollable tabs',
+  'Selection controls',
+  'Sliders',
+  'Snackbar',
+  'Tabs',
+  'Text fields',
+  'Time picker',
+  'Tooltips',
+  'Colors',
+  'Typography'
```

Comment:
I will add the comment now and change the gallery app to export this info in a separate PR.